src/__tests__/category/category.mutation.test.js

```diff
@@ -0,0 +1,201 @@
+                },
+                {
+                    lang: "ukr",
+                    value: "Тест"
+                },
+                ],
+                available: true,
+            }],
+            available: true
+        })).resolves.toBe(res);
+        
+        expect(categoryResolver.categoryMutation.addCategory('',{ category:{
+            name: [{
+                lang: "eng",
+                value: "Test"
+            },
+            {
+                lang: "ukr",
+                value: "Тест"
+            }
+            ],
+            categoryCode: "23131",
+            subcategories: [{
+                categoryCode: "22341",
+                name: [{
+                    lang: "eng",
+                    value: "Test"
+                },
+                {
+                    lang: "ukr",
+                    value: "Тест"
+                },
+                ],
+                available: true,
+            }],
+            available: true
+        }})).resolves.toBe(res);
+    })
+
+    test('should update category by id', async () => {
+        const res = await client.mutate({
+            mutation: gql`
+                mutation($categoryId: ID!) {       
+                    updateCategory(id: $categoryId, category: {
+                        name: [{
+                            lang: "eng",
+                            value: "Test"
+                        },
+                        {
+                            lang: "ukr",
+                            value: "Тест"
+                        }
+                        ],
+                    }){
+                        _id
+                        name {
+                            lang
+                            value
+                        }
+                        categoryCode
+                        subcategories {
+                            categoryCode
+                            name {
+                                lang
+                                value
+                            }
+                            available
+                        }
+                        available
+                    }
+                }
+            `,
+            variables: {
+                categoryId
+            }
+        })
+
+        expect(categoryService.updateCategory(categoryId, {
+            name: [{
+                lang: "eng",
+                value: "Test"
+            },
+            {
+                lang: "ukr",
+                value: "Тест"
+            }
+            ]
+        })).resolves.toBe(res);
+        
+        expect(categoryResolver.categoryMutation.updateCategory('',{ id: categoryId, category:{
+            name: [{
+                lang: "eng",
+                value: "Test"
+            },
+            {
+                lang: "ukr",
+                value: "Тест"
+            }
+            ]
+        }})).resolves.toBe(res);
+    })
+
+    test('should delete category', async () => {
```

Comment:
Please, have also cases where you provide invalid data to check how business validation works, for example, if you try to delete the category with id that doesn't exist inside database